src/index.js

```diff
@@ -1,42 +1,68 @@
 import { createPlugin } from 'docz-core'
 import * as path from 'path'
+import fs from 'fs-extra'
 import paths from './paths'
+import buildStorybook from './build-storybook'
 
 export const storybook = (opts = { }) => {
   const {
     // these options are intended to mimic the storybook cli options
-    configDir = paths.storybook.configDir
+    configDir = paths.storybook.configDir,
+    manual = false
   } = opts
 
   const storybookConfigPath = path.resolve(configDir, paths.storybook.config)
+  const storybookFiles = []
 
   return createPlugin({
-    setConfig: (config) => {
-      // TODO: since we're only given an entrypoint .storybook/config.js with a configure
-      // call to load stories on the client, we won't know which mdx files to create so
-      // we may have to inject entries dynamically on the client as part of the 'configure'
-      // process. after configure, we add all story kinds to the client-side entries?
+    setConfigPromise: async (config) => {
+      if (!manual) {
+        const storybook = await buildStorybook({ configDir })
+        console.log('storybook', JSON.stringify(storybook, null, 2))
```

Comment:
What level of logging should we be shooting for? This one looks like more of a debug log?

Reply:
Good call; we should remove most of this debugging stuff before we proceed too far w/ this module.

Looks like `docz` uses [signale](https://github.com/klauscfhq/signale) for its logging and we could follow suit with some optional verbose logging if we wanted.